packages/cx/src/_nested-list.scss

```diff
@@ -217,6 +233,35 @@ $button: #{$fd-cx-namespace}-button;
     }
   }
 
+  &__button {
+    @include fd-reset();
+    @include fd-flex-center();
+    @include fd-nested-list-focus();
+    @include fd-set-width(2.75rem);
+    @include fd-set-margin-left(0.5rem);
+
+    height: 100%;
+    cursor: pointer;
+    background: transparent;
+    border-radius: $fd-nested-list-item-border-radius;
+
+    [class*=sap-icon] {
+      @include fd-flex-center();
+      @include fd-set-width(1rem);
+      @include fd-set-height(1rem);
+
+      line-height: 1;
+
+      @include fd-rtl() {
+        -moz-transform: scale(-1, 1);
```

Comment:
Do we need all those vendor prefixes? isn't Autoprefixer does that by default?

Reply:
Autoprefixer does indeed that by default but in the past we faced issues with Nested List and had to manually add these (this was added as a fix).

Reply:
I removed them :)